ci/snapshot-publish/Jenkinsfile

```diff
@@ -13,6 +15,13 @@ if (currentBuild.getBuildCauses().toString().contains('BranchIndexingCause')) {
 }
 
 pipeline {
+	agent {
```

Comment:
Why not `agent any` instead of whole this change?

Reply:
I'm just trying to follow the example from the link in the description. 
Do you know of any working examples that I can take a look at? 

(I wonder whether we need that pipeline library here at all or if I should just go with `agent any`/`agent basic` and remove the pipeline library ... that was going to be my next "test" 😃  (https://eclipse-cbi.github.io/cbi-website/jiro-agent/index.html) )

Reply:
I'd also suggest for you to push to specific branch directly to this repo (or even modify Jenkinsfile online) - and trigger [pipeline](https://ci.eclipse.org/validation/job/jakarta-validation-api-publish-snapshot/) manually for that branch. And once you are happy with the outcome - do whatever you find appropriate for `main` based on that.